app/controllers/lessons_controller.rb

```diff
@@ -87,7 +90,10 @@ def update
         format.html { redirect_to @lesson, notice: 'Lesson was successfully updated.' }
         format.json { head :no_content }
       else
-        format.html { render action: "edit" }
+        format.html do
+          @venues = current_school.venues
+          render action: "edit"
+        end
```

Comment:
Thanks. I know it's tough to decide when to refactor in a messy code base, maybe this is a good opportunity,